src/components/Navigation/TopBar/Help/FeedbackForm/FeedbackForm.tsx

```diff
@@ -31,50 +44,129 @@ const FeedbackForm: FC<FeedbackFormProps> = ({ onClose, selectedType }) => {
   const [feedbackContent, setFeedbackContent] = useState<FeedbackContentType>({
     title: '',
     description: '',
-    optOutEmail: false,
+    anonymous: false,
   });
+  const [relevantRequestsIsSuccess, setRelevantRequestsIsSuccess] =
+    useState(false);
+
+  const {
+    mutateAsync: slackFileUpload,
+    isLoading: isFileUploadLoading,
+    isSuccess: isFileUploadSuccess,
+  } = useMutation(['slackFileUpload', feedbackContent], (formData: FormData) =>
+    PortalService.fileUpload(formData)
+  );
 
-  const { mutateAsync: slackFileUpload, isLoading: isFileUploadLoading } =
-    useMutation(['slackFileUpload', feedbackContent], (formData: FormData) =>
-      PortalService.fileUpload(formData)
-    );
+  const {
+    mutateAsync: slackPostMessage,
+    isLoading: isPostMessageLoading,
+    isSuccess: isPostMessageSuccess,
+  } = useMutation(['slackPostMessage', feedbackContent], (formData: FormData) =>
+    PortalService.postmessage(formData)
+  );
 
-  const { mutateAsync: slackPostMessage, isLoading: isPostMessageLoading } =
-    useMutation(['slackPostMessage', feedbackContent], (formData: FormData) =>
-      PortalService.postmessage(formData)
-    );
+  const {
+    mutateAsync: serviceNowIncident,
+    isLoading: isServiceNowLoading,
+    isSuccess: isServiceNowSuccess,
+    data: response,
+  } = useMutation(
+    ['serviceNowIncident', feedbackContent],
+    async (formData: FormData) => PortalService.createIncident(formData)
+  );
 
-  const { mutateAsync: serviceNowIncident, isLoading: isServiceNowLoading } =
-    useMutation(
-      ['serviceNowIncident', feedbackContent],
-      async (serviceNowDto: ServiceNowIncidentRequestDto) =>
-        PortalService.createIncident(serviceNowDto)
-    );
+  useEffect(() => {
+    const isSuccess: boolean[] = [];
```

Comment:
UseMutation still allows use of onSuccess, would that fit better here than doing this useEffect with isSuccess?

Reply:
For me it feels cleaner to do it in the useEffect. All the logic in a single place, and only one set state call in case all 3 requests are relevant.